dev-python/cadquery/cadquery-2.2_rc20220506.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_SINGLE_IMPL=1
+PYTHON_COMPAT=( python3_{8..10} )
+
+inherit distutils-r1
+
+DESCRIPTION="Python parametric CAD scripting framework based on OCCT"
+HOMEPAGE="https://cadquery.readthedocs.io"
+
+# The official CadQuery 2.1 tarball requires the obsolete OCCT 7.4.0, but
+# CadQuery 2.2 has yet to be officially released. We instead package a commit
+# known to work as expected with OCCT 7.5.2.
+MY_COMMIT="803a05e78c233fdb537a8604c3f2b56a52179bbe"
+
+#FIXME: Uncomment on bumping to the next stable release.
+# SRC_URI="https://github.com/CadQuery/cadquery/archive/refs/tags/${PV}.tar.gz"
+SRC_URI="https://github.com/CadQuery/cadquery/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+# Dependencies are intentionally listed in "conda/meta.yml" order. Due to its
+# Anaconda focus, "setup.py" currently fails to list dependencies.
+BDEPEND="
+	test? (
+		$(python_gen_cond_dep '
+			dev-python/pytest[${PYTHON_USEDEP}]
```

Comment:
We can actually drop the pytest dependency, it is already added by the `
distutils_enable_tests pytest` command.

Reply:
Deleted